corehq/ex-submodules/casexml/apps/case/tests/test_bugs.py

```diff
@@ -133,12 +133,11 @@ def testSubmitToDeletedCase(self):
         deleted_doc_type = 'CommCareCase-Deleted'
         [xform, [case]] = FormProcessorInterface.post_case_blocks([
             CaseBlock(create=True, case_id=case_id, user_id='whatever',
-                      update={'foo': 'bar'}).as_xml()
```

Comment:
hmm you're right i was just looking for a quick fix to not have to use `update_properties` to change the doc_type, but seems i could just delete the form and that'd have the same effect